src/views/evaluation/Evaluation.js

```diff
@@ -101,7 +91,7 @@ const Footer = function(option = {}){
     </footer>`
 }
 const EvaluationPage = function(option ={}){
-    return `${NAV(option.hContent)}${tableBody(option.tContent)}${Footer(option.fContent)}`
+    return `${NAV(option.headerContent, option.logoContent)}${tableBody(option.tContent)}${Footer(option.footerContent)}`
```

Comment:
Not sure if this is the same thing I got in the end, checking my code and this part is : 
`const EvaluationPage = function(option ={}){
    return `${NAV(option.headerContent, option.logoContent)}
    ${tableBody(option.tableContent, option.tableHeader)}
    ${Footer(option.footerContent)}`
}`

not sure what is up but I can say I changed that part for sure